cupy/_core/core.pxd

```diff
@@ -32,7 +32,7 @@ cdef class ndarray:
     cpdef bytes dumps(self)
     cpdef ndarray astype(self, dtype, order=*, casting=*, subok=*, copy=*)
     cpdef ndarray copy(self, order=*)
-    cpdef ndarray view(self, dtype=*)
+    cpdef ndarray view(self, dtype=*, typ=*)
```

Comment:
Seem that Cython does not allow using `type` in an argument list. Also, `numpy.ndarray.view` has the signature `ndarray.view([dtype][, type])`, so I suppose we should have (at most)two positional arguments for `cupy.ndarray.view` instead of keyword args.

https://numpy.org/doc/stable/reference/generated/numpy.ndarray.view.html

Reply:
Thanks for the clarification, I thought it was a typo at first.
How about naming it `array_class`? feels more descriptive :)

Reply:
Renamed!